README.md

```diff
@@ -40,33 +40,94 @@ annotations:
 You can test your TestGrid configuration locally with the `./hack/check-testgrid-config.sh`. Please open a PR for `ci-infra` repository for your new configuration. When it is merged the new configuration will be pushed to `gs://gardener-prow/testgrid/config` automatically and your jobs will become visible at [testgrid.k8s.io/gardener](https://testgrid.k8s.io/gardener) soon.
 
 
+## Combined kubeconfig for prow clusters and Gardener project
+
+The scripts from this repository rely on a combined `kubeconfig`. It contains two contextes for the prow clusters `gardener-prow-trusted`, `gardener-prow-build` and one for the Gardener project the clusters are created in.
+Please create your config based on this template:
+
+```yaml
+apiVersion: v1
+clusters:
+- cluster:
+    server: https://api.live.gardener.cloud.sap
+  name: garden-garden-ci
+- cluster:
+    certificate-authority-data: <<Get from your gardener project>>
+    server: <<Get from your gardener project>>
+  name: shoot--garden-ci--prow
+- cluster:
+    certificate-authority-data: <<Get from your gardener project>>
+    server: <<Get from your gardener project>>
+  name: shoot--garden-ci--prow-work
+contexts:
+- context:
+    cluster: garden-garden-ci
+    namespace: garden-garden-ci
+    user: oidc-login
+  name: garden-garden-ci
+- context:
+    cluster: shoot--garden-ci--prow-work
+    user: shoot--garden-ci--prow-work-token
+  name: gardener-prow-build
+- context:
+    cluster: shoot--garden-ci--prow
+    user: shoot--garden-ci--prow-token
+  name: gardener-prow-trusted
+current-context: gardener-prow-trusted
+kind: Config
+users:
+- name: oidc-login
+  user:
+    exec:
+      apiVersion: client.authentication.k8s.io/v1beta1
+      args:
+      - <<Get from your gardener project>>
+      command: kubectl
+- name: shoot--garden-ci--prow-token
+  user:
+    token: <<Get from your gardener project>>
+- name: shoot--garden-ci--prow-work-token
+  user:
+    token: <<Get from your gardener project>>
```

Comment:
Instead of using a non-personalized token, we could use `gardenlogin`, e.g. something like what is generated by `gardenctl`:
```yaml
clusters:
- cluster:
    certificate-authority-data: ...
    extensions:
    - extension:
        gardenClusterIdentity: sap-landscape-live
        shootRef:
          name: prow-work
          namespace: garden-garden-ci
      name: client.authentication.k8s.io/exec
    server: ...
  name: garden-garden-ci--prow-work-external
users:
- name: garden-garden-ci--prow-work
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1beta1
      args:
      - gardenlogin
      - get-client-certificate
      command: kubectl
      env: null
      interactiveMode: IfAvailable
      provideClusterInfo: true
```